src/cli/main.ts

```diff
@@ -27,22 +32,28 @@ import { findPackagesConfiguration } from "../input/findPackagesConfiguration";
 import { findESLintConfiguration } from "../input/findESLintConfiguration";
 import { findTSLintConfiguration } from "../input/findTSLintConfiguration";
 import { findTypeScriptConfiguration } from "../input/findTypeScriptConfiguration";
+import { findEditorConfiguration } from "../input/findEditorConfiguration";
 import { importer, ImporterDependencies } from "../input/importer";
 import { mergeLintConfigurations } from "../input/mergeLintConfigurations";
-import {
-    reportConversionResults,
-    ReportConversionResultsDependencies,
-} from "../reporting/reportConversionResults";
-import { converters } from "../rules/converters";
+import { reportSettingConversionResults } from "../reporting/reportSettingConversionResults";
+import { reportConversionResults } from "../reporting/reportConversionResults";
+import { ReportConversionResultsDependencies } from "../reporting/dependencies";
+import { converters as rulesConverters } from "../rules/converters";
+import { converters as settingsConverters } from "../settings/converters";
```

Comment:
I'm finding it hard to keep track of multiple `converters` in dependencies. What do you think of renaming them to always be referred to as `rulesConverters` and `settingsConverters`?

Reply:
Renamed files and exported converters, so that we can import and use them directly.